metadata-events/mxe-schemas/src/original/avro/com/linkedin/common/Deprecation.avsc

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Can you get rid of these Avro files? They are auto-generated.
`rm -rf metadata-events/mxe-schemas/src/original`